### ISSUE TITLE
Question about Reproducibility

### ISSUE THREAD

**Issue by Unkrible** [datamllab/Policy-GNN#7]
Hello, I have read your paper on KDD2020 and the code recently, it is an interesting work and outperforms all existing methods.
Congratulations!
But I have some problems about the reproducibility.
I tried to reproduce the results using `python train_cora.py` twice, but the result is even worse than vanilla GCN.
![image](https://user-images.githubusercontent.com/18095356/91387688-619ad000-e868-11ea-86b9-f30082afec06.png)

**Comment by lhenry15** [comment]
Here is the result I got on Citeseer and Cora from this version of code. I think the main problem might be the different version of library (numpy/pytorch/python version) you are using. The main reason might be the random seed setting In different version of library and python generate a different environment that is not compatible to our default hyper-parameter setting. Please note that, the environment of learning graph neural network is still very challenging, and we are also striving to make the whole framework as stable as possible. But in this version of code, what we can achieve now is tuning the hyperparameter to boost the performance on a relatively fixed and stable environment. Therefore, the random seed, library version as well as the python version should be strictly followed to reproduce the result. We are also trying to use alternative reinforcement learning algorithm with alternative reward function design (we think this should be a key point) which may probably help the whole framework more stable. Thank you for asking.

<img width="562" alt="citeseer" src="https://user-images.githubusercontent.com/6646890/91469711-5a1ffa80-e859-11ea-9f25-14b1c86a3346.png">
<img width="542" alt="cora" src="https://user-images.githubusercontent.com/6646890/91469713-5ab89100-e859-11ea-91e0-13716fe5af93.png">

**Comment by asenwu** [comment]
Hello, I am very interested in the work you do. But I have a few questions. First of all, I used the environment version you gave for testing，but the accuracy of the testing set was around 0.8, which is far from the best results in your article. Then, in the picture you gave, why is the accuracy of the testing set so much better than that of the validation set, and as the accuracy of the validation set increases, the accuracy of the test set decreases? Looking forward to your reply, thank you

**Comment by kaixin96** [comment]
Hi @lhenry15 , can you share the version of code that produce the result shown above (i.e. test acc >= 0.9)? The issue of different library versions can be avoid by providing a conda env yaml file. Thank you.

**Comment by dongwu92** [comment]
I've create a conda env with exactly the same versions of python libraries, while still getting around 0.4 or 0.8 acc randomly on cora...seems weird

**Comment by Huangzhizhong123** [comment]
python3.8
numpy==1.16.4  
torch-cluster==1.4.5  
torch-scatter==1.4.0  
torch-sparse==0.4.3  
torch-geometric==1.3.2  
torch==1.8.0
I set seed as 53 and get accuracy :


![acc](https://user-images.githubusercontent.com/84257756/119221699-1675cb00-bb23-11eb-9c32-ebc714052372.PNG)
